output_frame_id: base_link
output_topic: '/concat_rslidar_points'
input_topics: ['/left_rslidar/rslidar_points', '/right_rslidar/rslidar_points']